code/modules/client/preference/loadout/loadout.dm

```diff
@@ -0,0 +1,74 @@
+var/list/loadout_categories = list()
+var/list/gear_datums = list()
+
+/datum/loadout_category
+	var/category = ""
+	var/list/gear = list()
+
+/datum/loadout_category/New(cat)
+	category = cat
+	..()
+
+/hook/startup/proc/populate_gear_list()
+	//create a list of gear datums to sort
+	for(var/geartype in subtypesof(/datum/gear))
+		var/datum/gear/G = geartype
+
+		var/use_name = initial(G.display_name)
+		var/use_category = initial(G.sort_category)
+
+		if(G == initial(G.subtype_path))
```

Comment:
Now that's a useful trick for filtering categorization entries out. Going to remember that one.
Just wondering, will an -instance- of ```/datum/gear``` really match the -type- ```/datum/gear``` or should it actually be ```G.type == initial(G.subtype_path)```?